sky/engine/core/painting/RRect.dart

```diff
@@ -1,32 +1,234 @@
 part of dart_ui;
 
-// A rounded rectangle.
+// A rounded rectangle with the same radii for all four corners.
 class RRect {
-  RRect();
+  RRect._();
```

Comment:
I don' t think it can be (same problem with Rect), because the data is stored in a mutable Float32List. (Which should probably be a Float64List? I don't really understand what we're doing there. I assume it's to do with how efficiently this data can be passed to the C++ side?)